src/html5_parser/node.rs

```diff
@@ -38,7 +38,7 @@ pub enum NodeData {
 
 /// Id used to identify a node
 #[derive(Clone, Copy, Debug, Default, Eq, Hash, PartialEq, Display)]
-pub struct NodeId(pub(crate) usize);
+pub struct NodeId(usize);
```

Comment:
I took a look at this, and it wasn't hard to make private again.  I did this just to illustrate how this can be done, but it's not a big deal for it to be crate-public.